src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder result = new StringBuilder();
+        int[] arrayResultInt = new int[names.length];
+        List<String> list = new ArrayList<>(List.of(names));
+        LocalDate startDate = LocalDate.parse(dateFrom, formatter);
+        LocalDate endDate = LocalDate.parse(dateTo, formatter);
+        for (String s : data) {
```

Comment:
do not use short naming
```suggestion
        for (String dataPerDay : data) {
```